companies/porkbun.json

```
@@ -0,0 +1,26 @@
+{
+    "slug": "porkbun",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Porkbun",
+    "address": "21370 SW Langer Farms Parkway, Suite 142-429\nSherwood, OR 97140, US",
+    "email": "abuse@porkbun.com",
+    "web": "https://porkbun.com",
+    "sources": [
+        "https://porkbun.com",
+        "https://github.com/datenanfragen/data/pull/1324"
+    ],
+    "required-elements": [
+        {
+            "desc": "Account E-Mail Address",
```

```suggestion
            "desc": "Email",
```
naming according to #720 preferred?